src/ray/raylet/format/node_manager.fbs

```diff
@@ -157,6 +157,8 @@ table FetchOrReconstruct {
   object_ids: [string];
   // Do we only want to fetch the objects or also reconstruct them?
   fetch_only: bool;
+  // False for direct actor call tasks.
```

Comment:
More info about how this is actually handled would be more helpful

Reply:
Ah right, clarified.